mongo/integration/sessions_test.go

```diff
@@ -330,6 +330,23 @@ func TestSessions(t *testing.T) {
 	})
 }
 
+func TestSessionsStandalone(t *testing.T) {
+	standaloneOpts := mtest.NewOptions().MinServerVersion("3.6").Topologies(mtest.Single)
```

Comment:
This should include `CreateClient(false)`.

Reply:
`mt.RunOpts` will run a sub-test that creates a Client for itself. IIUC, The top-level `mtest.T` instance does not need it's own Client.

Reply:
done